fe/fe-core/src/main/java/com/starrocks/common/proc/CurrentQueryStatisticsProcDir.java

```diff
@@ -57,6 +57,7 @@ public class CurrentQueryStatisticsProcDir implements ProcDirInterface {
     private static final Logger LOG = LogManager.getLogger(CurrentQueryStatisticsProcDir.class);
     public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
             .add("StartTime")
+            .add("CustomQueryId")
```

Comment:
I suggest adding CustomQueryId behind field `Warehouse`, then the result can be back compatible.

Reply:
Modified.